src/collection-facets/more-facets-pagination.ts

```diff
@@ -0,0 +1,163 @@
+import { css, CSSResultGroup, html, LitElement } from 'lit';
+import { customElement, property, query, state } from 'lit/decorators.js';
+import arrowLeftIcon from '../assets/img/icons/arrow-left';
+import arrowRightIcon from '../assets/img/icons/arrow-right';
+
+@customElement('more-facets-pagination')
+export class MoreFacetsPagination extends LitElement {
+  @property({ type: Number }) paginationSize?: any | undefined;
+
+  @property({ type: Number }) step?: any | undefined;
+
+  @query('.page-numbers') private pageNumberElement!: HTMLElement;
+
+  @state() currentPage = 1;
+
+  firstUpdated() {
+    this.buildPagination();
+  }
+
+  private buildPagination() {
+    this.pageNumberElement.innerHTML = '';
+
+    // Calculate the startPage and endPage.
+    let startPage = this.currentPage - this.step;
+    let endPage = this.currentPage + this.step;
```

Comment:
will be nice to isolate this code so that we can create a unit test against the logic/computational part

Reply:
I want to split out the unit test part in another ticket.